meta.json

```diff
@@ -1,25 +1,30 @@
 {
-  "module_id": "erh:viamrtsp",
-  "visibility": "public",
-  "url": "https://github.com/erh/viamrtsp",
+  "module_id": "seanorg:seanviamrtsp",
```

Comment:
Does this need to change back at this point or still a todo? Since Nick just approved this pr.

Reply:
Yep I will triple check before merging. I have it like this right now in case we need to make more test releases in my org. It's easy to revert I just apply this diff basically https://github.com/erh/viamrtsp/pull/32/commits/44df9a4a6e85d61ce505f5740d0cd0bf073a203f